src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,36 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private int size;
+
+    private K[] keys = (K[]) new Object[10];
+    private V[] values = (V[]) new Object[10];
```

Comment:
```suggestion
    private static final int MAX_ELEMENTS_NUMBER = 10;
    private int size;
    private K[] keys;
    private V[] values;
    
    public StorageImpl() {
        keys = (K[]) new Object[MAX_ELEMENTS_NUMBER];
        values = (V[]) new Object[MAX_ELEMENTS_NUMBER];
    }
```
better initialize arrays in constructor;
create constants for magic numbers;